src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -82,9 +132,25 @@ export class TablePaginationControls extends React.Component<ITablePaginationCon
                     <Button key="prevPageBtn" disabled={!!this.props.previousPageDisabled} onClick={this.props.onPreviousPageClick}>
                         {this.props.previousButtonContent}
                     </Button>
-                    <Button key="textBetweenButtons" className={styles["auto-cursor"]} disabled={true}>
-                        {this.props.textBetweenButtons}
-                    </Button>
+                    <If condition={!this.textBoxActive}>
+                        <Then>
+                            <Button key="textBetweenButtons" className={styles["auto-cursor"]} disabled={!this.props.pageNumberEditable}
```

Comment:
I tried it on heroku deploy.  This is an Ethan question but I would vote for just the page number to become editable so you can still see the upper page bound.